frontend/src/Indexer/Edit/EditIndexerModalContent.js

```diff
@@ -116,6 +117,18 @@ function EditIndexerModalContent(props) {
                 />
               </FormGroup>
 
+              <FormGroup>
```

Comment:
Shouldn't this be disabled for non-torrent indexers?

Reply:
You're absolutely right! I completely missed this. Fixed in [ac1181bbd93f57177eda05d83f38f9fde17ea286](https://github.com/Deathspike/Prowlarr/commit/ac1181bbd93f57177eda05d83f38f9fde17ea286). Thank you!